### ISSUE TITLE
Correct minor details in data and metadata of OpenMRG data transformation

### ISSUE THREAD

**Issue by cchwala** [OpenSenseAction/OPENSENSE_sandbox#67]
Based on feedback from the creators of OpenMRG we need to correct some minor details in the NetCDF file and in the metadata. The transformation of the raw data did not show any issues.

Here is the info regarding required corrections (added as quotes) with comments from my side:

**NC file generally:**
> - Global attributes: probably the NA values should be filled with something according to OpenSENSE standard. Erlend you should add your name as author of this specific file, especially to the example file in the sandbox/github

Fill values should be handled automatically by NetCDF if they are specified correctly. We found that the OPENSENSE conventions are not in line with the NetCDF standard for file (see https://github.com/OpenSenseAction/OS_data_format_conventions/issues/10). I suggest to do it the way the NetCDF requires it.

@eoydvin Should definitely add his name and/or add info on which code was used to do the transformation. Actually there is a mixture of code from different authors that is currently used, I guess.

> - "history" or "comment" at least should have info about what Erlends script does (beyond the original OpenMRG files), especially when you add interpolation, and resampling to 1min resolution in the sandbox example file.

Yes.

> - "coordinates" does not make sense to me, but
> - Note I did not check compliance with OpenSENSE in terms of data format/naming/attributes etc. https://github.com/OpenSenseAction/OS_data_format_conventions/blob/main/netCDF_CML.adoc

@eoydvin I think you and @maxmargraf did check that it aligns with the conventions. Please check again, but I guess the coordinates are correct now.

**Metadata:**
> - I believe the frequency is provided in GHz but the unit in the nc file says MHz, this should be corrected to whatever is the OpenSENSE standard.

Yes, frequency values must be stored in MHz. This is what our conventions require.

> - units missing for length and coordinates (seems to be km and decimal degrees I guess, check what the OpenSENSE standard is and potentially convert).

Yes this should be specified clearly in the file. The convention says that length is in meters. But for coordinates it is not specified what type of degrees are used. Decimal degrees makes most sense, since degrees with arcsec etc. is hard to parse. I suggest to make an update of the conventions to clearly state that we want decimal degrees.

> - sublink_id: maybe it would make more sense to call them "A" and "B" to link closer with the original data, unless it is an OPENSENSE convention to do it like this.

There is no requirement for what to name the sublinks, i.e. what to use as sublink_id. If I understand the comment correctly, this would mean one sublink should be "AB" and the other "BA", but I am not sure.

Anyway, what could be good is to encode the original sublink names in the `cml_id`, e.g. as `001_002` to indicate that the sublink `001` and `002` from the original file have been combined into one CML with these to sublinks. That is not crucial, though.

> Data (RSL, TSL)
> - no errors found, only rounding differences 

Good 🎉

**Comment by cchwala** [comment]
@eoydvin and @maxmargraf If you work on this, which would be highly appreciated, I suggest to track things that need to be updated in the WG1 data format conventions in one or more separate issues there.

**Comment by jcmand** [comment]
Thanks for working on this!

**Comment by cchwala** [comment]
fixed via #68
